src/pages/developers/sdk/web/media/index.md

```diff
@@ -428,7 +428,7 @@ For example:
             "content_duration": "120000",
             "content_id": "1234567",
             "content_type": "Video",
-            "stream_type": "OnDemand",
+            "stream_type": "LiveStream",
```

Comment:
any reason for this change from OnDemand to LiveStream?

Reply:
customers don't know the supported stream types without looking into the sdk code. this code snippet demostrates the second possible value LiveStream for stream_type.

Reply:
i see your point.  to provide another example and to include the other enum.  it's now addressed in your change above saying it's either OnDemand or LiveStream, but having this is helpful.